assets/css/main.css

```diff
@@ -1,11 +1,10 @@
 /* =============================================================================
 HTML5 CSS Reset Minified - Eric Meyer
```

Comment:
I always prefer to include CSS resets as a separate asset, but I suppose it's fine being here.